pkg/core/transaction/witness_test.go

```diff
@@ -36,3 +36,16 @@ func TestWitnessSerDes(t *testing.T) {
 	require.Error(t, testserdes.DecodeBinary(bin1, exp))
 	require.Error(t, testserdes.DecodeBinary(bin2, exp))
 }
+
+func TestWitnessCopy(t *testing.T) {
+	original := &Witness{
+		InvocationScript:   make([]byte, 64),
+		VerificationScript: make([]byte, 32),
+	}
+
+	copyWitness := original.Copy()
+	require.Equal(t, *original, copyWitness)
+
+	copyWitness.VerificationScript = make([]byte, MaxVerificationScript+1)
```

Comment:
And here instead just change some byte of the original witness, like `original.InvocationScript[0] = 0x05`.